src/tools/rbd/ArgumentTypes.h

```diff
@@ -81,7 +81,9 @@ static const std::string PRETTY_FORMAT("pretty-format");
 static const std::string VERBOSE("verbose");
 static const std::string NO_ERROR("no-error");
 
-static const std::string DELAY("delay");
+static const std::string EXPIRES_AT("expires-at");
+static const std::string OLDER_THAN("older-than");
```

Comment:
Nit: perhaps for clarity the option should be ```expired-before``` since it's based on the expiration time and not the deletion time?